fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/api/OCFLObjectSession.java

```diff
@@ -104,4 +106,12 @@
      */
     void close() throws PersistentStorageException;
 
+
+    /**
+     * Return the list of versions associated with this OCFL Object in chronological order.
+     * @return The list of versions
+     * @throws PersistentStorageException
```

Comment:
Needs description

Reply:
will do.

Reply:
I'll clarify that the list is the list of immutable versions.